composer.json

```
@@ -19,9 +19,9 @@
     "doctrine/doctrine-bundle": "^1.10|^2.0",
     "doctrine/orm": "^2.7",
     "ramsey/uuid-doctrine": "^1.5",
-    "symfony/workflow": "^4.4|^5.2",
-    "symfony/validator": "^4.4|^5.2",
-    "symfony/framework-bundle": "^4.4|^5.2"
+    "symfony/workflow": "^6.0",
+    "symfony/validator": "^6.0",
+    "symfony/framework-bundle": "^6.0"
```

```suggestion
    "symfony/workflow": "^4.4|^5.4|^6.0",
    "symfony/validator": "^4.4|^5.4|^6.0",
    "symfony/framework-bundle": "^4.4|^5.4|^6.0"
```

Graag alle Symfony ondersteunde versies accepteren.